TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -227,18 +227,22 @@ class TestNSHTTPCookieStorage: XCTestCase {
         storage.setCookie(testCookie)
         XCTAssertEqual(storage.cookies!.count, 3)
         var destPath: String
+        //let bundleName = Bundle.main.object(forInfoDictionaryKey: "CFBundleDisplayName") as! String
+        //TODO: Above api returns nil on Linux when this code is introduced. Hence, using the below workaround.
```

Comment:
On linux, NSBundle.infoDictionary:

```
 infoDictionary: [String : Any]? {
        let cfDict: CFDictionary? = CFBundleGetInfoDictionary(_bundle)
```

I printed the cfDict and saw the below values:

Optional<Dictionary<AnyHashable, Any>>
Key: CFBundleShortVersionString Value: 1.0
Key: CFBundleDevelopmentRegion Value: en
Key: NSHumanReadableCopyright Value: Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
Key: CFBundleInfoDictionaryVersion Value: 6.0
Key: **CFBundleName** Value: TestFoundation
Key: CFBundleIdentifier Value: org.swift.TestFoundation
Key: CFBundleVersion Value: 1
Key: CFBundleNumericVersion Value: 16809984
Key: NSPrincipalClass Value: 
Key: CFBundleExecutable Value: TestFoundation
Key: CFBundlePackageType Value: APPL
Key: CFBundleSignature Value: ????

It does have the Bundle name and other details, whereas the cast 
`return dict as! [String : Any] `
fails. Thats why it looks like a bridging issue to me on Linux.

I’ve removed the commented code.